packages/SwingSet/package.json

```diff
@@ -28,6 +28,7 @@
   },
   "devDependencies": {
     "@agoric/install-metering-and-ses": "^0.2.3",
+    "@agoric/ses-ava": "^0.1.0",
```

Comment:
You also list this in dependencies. Don't dependencies imply devDependencies?

Reply:
Ah, I thought yarn would remove automatically rather than put it in both. I only intend for it to be in dependencies.